README.md

```diff
@@ -54,31 +54,31 @@ We've added Storybook pages to show how each component can be used.
 Upon a new component being added to this table run 'npm run update-readme' for boilerplate code. Please replace the three variables with the corresponding information as needed. Also, store screenshots of components in the 'screenshots' folder.
 ### <b>Please do not add information below this table, the 'npm run update-readme' will not work properly if the table is not last.
 ---
```

Comment:
Locally the table is showing a syntax error for all table rows should have borders (a pipe) at the start and end. The preview of this seems okay though.